lib/block-supports/typography.php

```diff
@@ -455,16 +459,28 @@ function gutenberg_get_typography_font_size_value( $preset, $should_use_fluid_ty
 		return $preset['size'];
 	}
 
+	/*
+	 * Backwards compatibility since 6.5.
+	 * If the settings argument is a boolean, it's a fluid typography override.
+	 */
+	if ( is_bool( $settings ) ) {
+		$settings = array(
+			'typography' => array(
```

Comment:
This bit of inelegant magic ensures backwards compatibility.

Searching https://wpdirectory.net/ I don't see anyone using the second bool argument for `wp_get_typography_font_size_value()`